packages/select/src/common/itemListRenderer.ts

```diff
@@ -60,6 +60,11 @@ export interface IItemListRendererProps<T> {
      * to the owner component's `itemRenderer` prop.
      */
     renderItem: (item: T, index: number) => JSX.Element | null;
+
+    /**
+     * The rendered createIem.
+     */
+    createItemView: JSX.Element | null;
```

Comment:
let's make this a function, as suggested in https://github.com/palantir/blueprint/issues/4031#issuecomment-667985872

```suggestion
    renderCreateItem: () => JSX.Element | null;
```